torchci/clickhouse_queries/nightly_jobs_red_past_day/query.sql

```diff
@@ -1,20 +1,18 @@
--- !!! Query is not converted to CH syntax yet.  Delete this line when it gets converted
 SELECT
-  COUNT(*) COUNT,
-  job.name
+  COUNT(*) AS COUNT,
+  job.name as name
 FROM
-  commons.workflow_job job
-  JOIN commons.workflow_run workflow on workflow.id = job.run_id
-  JOIN push on workflow.head_commit.id = push.head_commit.id
+  workflow_job job
+  JOIN workflow_run workflow ON workflow.id = job.run_id
+  join push on push.head_commit.'id' = workflow.head_commit.'id'
 WHERE
   job.name NOT LIKE '%generate-matrix%'
   AND job.name NOT LIKE '%unittests%'
   AND workflow.name NOT IN ('cron', 'Bandit', 'tests')
   AND push.ref = 'refs/heads/nightly'
-  AND push.repository.owner.name = 'pytorch'
-  AND push.repository.name = :repo
-  AND job.conclusion in ('failure', 'timed_out', 'cancelled') 
-  AND job._event_time >= CURRENT_DATE() - INTERVAL 1 DAY
+  AND push.repository.'owner'.'name' = 'pytorch'
+  AND push.repository.'name' = {repo: String }
+  AND job.conclusion IN ('failure', 'timed_out', 'cancelled')
+  AND job.completed_at >= today() - 1
```

Comment:
Minor note that today() doesn't include hour so you're getting more than 24 hours of data here, don't know if it matters though

Reply:
I think its ok we just want to have a list that failed lately